### PR TITLE
[ES-7733] Update Readme

### PR DESCRIPTION
**Pull request by Sisyphus25**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,7 +146,91 @@ var response = await psClient.productSearchById(productId, parameters);
 
 > The request parameters for this API can be found at [ViSenze Documentation Hub](https://ref-docs.visenze.com/reference/visually-similar-api).
 
-## 4. Search results
+## 4. Advanced search
+
+### 4.1 Automatic object detection
+
+The Search API is smart to detect the objects present in the query image and suggest the best matched product type to run the search on.
```
Comment (wkurniawan07) [inline README.md]: "is smart to" sounds a bit weird. Suggest:
```suggestion
The Search API is able to detect the objects present in the query image and suggest the best matched product type to run the search on.
```

File: README.md [hunk 2]
```diff
@@ -218,9 +337,13 @@ psClient.sendEvents('transaction',
 );
 ```
 
-### 5.3 Getting event and tracking parameters
+### 6.3 Event parameters
+
+For list of all event parameter and their explanations, please refer to this [doc](https://ref-docs.visenze.com/reference/event-parameters)
```
Comment (wkurniawan07) [inline README.md]:
```suggestion
For list of all event parameter and their explanations, please refer to [this doc](https://ref-docs.visenze.com/reference/event-parameters)
```

**Review by wkurniawan07 — COMMENTED** [review]
Just some copy suggestions, looks fine otherwise

File: README.md [hunk 3]
```diff
@@ -5,12 +5,42 @@
...
+  - [2. Set up](#2-set-up)
+  - [3. APIs](#3-apis)
+    - [3.1 Search by image](#3.1-search-by-image)
+    - [3.2 Recommendations](#3.2-recommendations)
+  - [4. Advanced search](#4-advanced-search)
+    - [4.1 Automatic object detection](#4-advanced-search)
+    - [4.2 Filters and text filters](#4-filters-and-text-filters)
+    - [4.3 Facets](#4-facets)
+    - [4.4 Attributes retrieval](#4-attributes-retrieval)
+  - [5. Search results](#4-search-results)
```
Comment (bo-git) [inline README.md]: is the reference correct? #5?

Comment (Sisyphus25) [reply README.md]: Fixed

**Review by srfication — APPROVED** [review]

**Pull request merged**